src/App.js

```diff
@@ -1,24 +1,63 @@
-import logo from './logo.svg';
+import react, { useState } from 'react';
+
+import foodsJSON from "./foods.json";
 import './App.css';
+import FoodBox from './components/FoodBox';
+import AddFoodForm from './components/AddFoodForm.jsx';
+import Searchbar from './components/Searchbar';
+
+
 
 function App() {
+  const foodArr = [...foodsJSON]
+  const [foodList, setFoodList] = useState(foodsJSON);
+  const [showFood, setShowFood] = useState(foodsJSON)
+  
+
+  const addFood = (food) => {
+    const foodListCopy = [...foodList, food];
+    setFoodList(foodListCopy)
```

Comment:
Your `addFood()` function is not working because you're only updating `foodList` with `setFoodList` you need to update `showFood` as well, because that's the variable that you're using to loop through and display all the foods.
```
  const addFood = (food) => {
    const foodListCopy = [food, ...foodList];
    setFoodList(foodListCopy);
    setShowFood(foodListCopy);
  };
```